tests/unit/test_storage_service.py

```
@@ -23,55 +22,72 @@ async def test_save_and_load_valid_key(storage_service):
+        InvalidKey, match=r"Key cannot contain '\.\.' or start with '/'"
+    ):
         await storage_service.save(key, data)
 
+
 @pytest.mark.asyncio
 async def test_load_invalid_key_dot_dot(storage_service):
-        with pytest.raises(InvalidKey, match=r"Key cannot contain '\\.\.' or start with '/' "):\
+    with pytest.raises(
+        InvalidKey, match=r"Key cannot contain '\\.\.' or start with '/' "
+    ):
         await storage_service.load(key)
```

Ruff (0.14.5)

Undefined name `key`

(F821)